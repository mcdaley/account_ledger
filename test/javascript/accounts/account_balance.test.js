//-----------------------------------------------------------------------------
// test/javascripts/accounts/account_balance.test.js
//-----------------------------------------------------------------------------
import React          from 'react'
import AccountBalance from '../../../app/javascript/components/accounts/account_balance'
import renderer       from 'react-test-renderer'

it('renders correctly', () => {
  const tree = renderer
    .create(<AccountBalance credits="200.00" debits="50.00" balance="150.00" />)
    .toJSON()
  expect(tree).toMatchSnapshot()
});