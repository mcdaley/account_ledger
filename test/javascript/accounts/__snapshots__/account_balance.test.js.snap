// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row account-balances-row"
>
  <div
    className="col-4"
  >
    <div
      className="card"
    >
      <div
        className="card-header"
      >
        Credit
      </div>
      <div
        className="card-body account-card-body"
      >
        <h5
          className="card-title account-card-title"
        >
          $200.00
        </h5>
      </div>
    </div>
  </div>
  <div
    className="col-4"
  >
    <div
      className="card"
    >
      <div
        className="card-header"
      >
        Debit
      </div>
      <div
        className="card-body account-card-body"
      >
        <h5
          className="card-title account-card-title"
        >
          $50.00
        </h5>
      </div>
    </div>
  </div>
  <div
    className="col-4"
  >
    <div
      className="card"
    >
      <div
        className="card-header"
      >
        Balance
      </div>
      <div
        className="card-body account-card-body"
      >
        <h5
          className="card-title account-card-title"
        >
          $150.00
        </h5>
      </div>
    </div>
  </div>
</div>
`;
